<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePlayer</class>
 <widget class="QWidget" name="SimplePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimplePlayer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>暂停</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resumeBtn">
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startRecordBtn">
       <property name="text">
        <string>开始录制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopRecordBtn">
       <property name="text">
        <string>停止录制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="RtspStreamPlayer" name="player">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RtspStreamPlayer</class>
   <extends>QOpenGLWidget</extends>
   <header>RtspStreamPlayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
